dist/rpm/os-autoinst.spec

```diff
@@ -73,20 +73,31 @@ Source0:        %{name}-%{version}.tar.xz
 %else
 %define python_style_requires %{nil}
 %endif
+%ifnarch ppc ppc64 ppc64le s390x
+%bcond_with ocr
+%else
+%bcond_without ocr
+%endif
+%if %{with ocr}
+# The following line is generated from dependencies.yaml
+%define ocr_requires tesseract-ocr tesseract-ocr-traineddata-english
+%else
+%define ocr_requires %{nil}
+%endif
 # The following line is generated from dependencies.yaml
 %define test_base_requires %main_requires cpio icewm perl(Benchmark) perl(Devel::Cover) perl(FindBin) perl(Pod::Coverage) perl(Test::Fatal) perl(Test::Mock::Time) perl(Test::MockModule) perl(Test::MockObject) perl(Test::MockRandom) perl(Test::Mojo) perl(Test::Most) perl(Test::Output) perl(Test::Pod) perl(Test::Strict) perl(Test::Warnings) >= 0.029 procps python3-setuptools qemu >= 4.0 qemu-tools qemu-x86 xorg-x11-Xvnc xterm xterm-console
 # The following line is generated from dependencies.yaml
 %define test_version_only_requires perl(Mojo::IOLoop::ReadWriteProcess) >= 0.28
 # The following line is generated from dependencies.yaml
-%define test_requires %build_requires %spellcheck_requires %test_base_requires %yamllint_requires perl(Inline::Python) python3-Pillow-tk tesseract-ocr tesseract-ocr-traineddata-english
+%define test_requires %build_requires %ocr_requires %spellcheck_requires %test_base_requires %yamllint_requires perl(Inline::Python) python3-Pillow-tk
 # The following line is generated from dependencies.yaml
 %define devel_requires %python_style_requires %test_requires ShellCheck perl(Code::TidyAll) perl(Devel::Cover) perl(Devel::Cover::Report::Codecov) perl(Perl::Tidy) perl(Template::Toolkit)
 %define s390_zvm_requires /usr/bin/xkbcomp /usr/bin/Xvnc x3270 icewm xterm xterm-console xdotool fonts-config mkfontdir mkfontscale
 BuildRequires:  %test_requires %test_version_only_requires
 # For unbuffered output of Perl testsuite, especially when running it on OBS so timestamps in the log are actually useful
 BuildRequires:  expect
 Requires:       %main_requires
-Recommends:     tesseract-ocr
+Recommends:     %ocr_requires
```

Comment:
hm, error message:
```
error: line 100: Empty tag: Recommends:
```

Reply:
Yes, the whole `Recommends:` needs to be wrapped. Additionally, the macros for "conditional builds" work the other way around (see https://rpm-software-management.github.io/rpm/manual/conditionalbuilds.html). It should be fixed now.